pytket/pytket/qasm/qasm.py

```diff
@@ -242,7 +242,7 @@ def _extract_reg(var: Token) -> tuple[str, int]:
 
 
 def _load_include_module(
-    header_name: str, flter: bool, decls_only: bool
+    header_name: str, filter_bool: bool, decls_only: bool
```

Comment:
This isn't actually a typo, it's to avoid a name clash with a reserved keyword, and I don't think the replacement is an improvement.

Reply:
Gone with `do_filter` instead?